### ISSUE TITLE
ascii chars above 'g' are missing

### ISSUE THREAD

**Issue by sjoerdtimmer** [Artekit/artekit_vga#1]
At first I thought this was a glitch in my own code but in font5x7.c there are 0x00's everywhere from 'h' onwards. Is there a technical reason for this that I'm missing?

**Comment by Artekit-Dev** [comment]
Yes. I am afraid the list is not complete. Lowercase chars starting from 'h' are missing.

**Comment by sjoerdtimmer** [comment]
I have fixed my local install by converting the 5x7 font from here to your format: https://github.com/PaulStoffregen/LedDisplay/blob/master/font5x7.h

If you want I can share that with you via a pull request. Not sure how that works with licences.

**Comment by Artekit-Dev** [comment]
Sure. This project is under GPLv2.
